app/models/attachment.rb

```diff
@@ -7,7 +8,11 @@ class Attachment
   embedded_in :object, polymorphic: true
 
   def as_json *args
-    result = attributes.symbolize_keys.merge asset: asset.as_json
+    result = attributes.symbolize_keys.except(:_id, :_type, :asset_id).merge(
```

Comment:
I think I can see the change in the thing that consumes this ([here](https://github.com/withassociates/slices/pull/116/files#diff-771bae8f8b744e9e061f0b83820792b9R93), so I guess it's ok. But this came in as part of making sure they were all strings, so wasn't sure why the name had to change too. And there is a [spec](https://github.com/withassociates/slices/blob/master/spec/models/slice/write_attributes_spec.rb#L39) which looks like it needs updating..? Except it's not failing, so maybe I'm missing something.